cmd/virtual-kubelet/root/root.go

```diff
@@ -114,10 +85,12 @@ func runRootCommand(ctx context.Context, c *Opts) error {
 
 		Namespace: c.KubeletNamespace,
 		NodeName:  c.NodeName,
+		NodeIP:    os.Getenv("VKUBELET_POD_IP"),
```

Comment:
Should we make sure `VKUBELET_POD_IP` is set?

Reply:
Would postpone to a subsequent refactor, addressing #956